core/src/main/java/bisq/core/btc/wallet/BtcWalletService.java

```diff
@@ -487,8 +499,6 @@ public Transaction completePreparedBsqTx(Transaction preparedBsqTx,
         int sigSizePerInput = 106;
         // typical size for a tx with 2 inputs
         int txVsizeWithUnsignedInputs = 203;
-        // If useCustomTxFee we allow overriding the estimated fee from preferences
-        Coin txFeePerVbyte = useCustomTxFee ? getTxFeeForWithdrawalPerVbyte() : feeService.getTxFeePerVbyte();
```

Comment:
Looks like custom fee was always used when calling this method anyway.

Reply:
Yes, unless the `preferences.isUseCustomWithdrawalTxFee()` was false.
Only if the `useCustomTxFee` param was `false` (it never was), would the fee service would be used.

Reply:
The name of useCustomTxFee was badly choosen. The intention was that for certain important transactions we want to use always the fee service (like in trade txs). I think in some BSQ txs that was the case earlier as well but got changed later as BSQ txs do not cause harm to other peers in case the fee was too low set.

So intention was to have an option to enforce usage of the fee service independent of what user has set in preferences. The false case would have done that. The true case will use whatever the user has set in preferences.